src/components/CmsZone.vue

```diff
@@ -411,29 +434,6 @@ export default class CmsZone extends Vue {
 }
 </script>
 
-<style>
-.scrollable {
```

Comment:
just to confirm my understanding: we're removing these because we're putting these zones inside scrollable containers, instead of having the zone itself be scrollable?

Reply:
The previous implementation relied on the class names to accomplish various things, with the move to intersection observer this is no longer necessary. Turns out these slides were not used on FC at all, and there's already a copy of them in freshebt-vue. Axed it, since they are no longer a hard dependency for any code in vue-phoenix.